src/styles/_catalog.scss

```
@@ -1,22 +1,57 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  gap: $gap;
+  justify-content: center;
+  align-content: center;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+
+    transition: transform $trans-speed;
+
+    &:hover {
+      transform: scale(1.2);
+      color: $color-grey;
+    }
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    width: getCatalogWidth($cardWidth, 2);
+
+    grid-template-areas: "card card"
+      "card card";
   }
+}
```

```suggestion
.catalog {

      /* styles */

  @media (min-width: 488px) {
      grid-template-columns: 1fr 1fr;
      grid-template-rows: 1fr 1fr;
      width: getCatalogWidth($cardWidth, 2);
  
      grid-template-areas: "card card"
        "card card";
     }
}
```
If you put your medias inside the block that they are used for, it will be easier for you to find them and more clear tell what are for.